### ISSUE TITLE
SDR branch causing chain sync anomalies

### ISSUE THREAD

**Issue by mgoelzer** [filecoin-project/lotus#4698]
**Describe the bug**
I deleted my .lotus directory, built the SDR branch (per [@magik6k's instructions here](https://filecoinproject.slack.com/archives/C0179RNEMU4/p1603810230225400)) and then started syncing to the chain.  Chain sync does not work; instead it ends up in this "unknown" state:

```
$ lotus sync wait
Worker: 0; Base: 0; Target: 0 (diff: 0)
State: <unknown: 0>; Current Epoch: 0; Todo: 0
Validated 0 messages (0 per second)
```

**To Reproduce**
1.  `rm -rf .lotus`
2.   Follow @magik6k [compile instructions here](https://filecoinproject.slack.com/archives/C0179RNEMU4/p1603810230225400)
3.  `nohup lotus daemon --import-snapshot https://fil-chain-snapshots-fallback.s3.amazonaws.com/mainnet/minimal_finality_stateroots_latest.car > ~/lotus-daemon.log &`
4.   Wait for the car import to finish
6.  `lotus daemon stop`
7.  Wait for graceful termination
8.  Restart lotus like this:  `nohup lotus daemon > ~/lotus-daemon.log &`
9.  `lotus sync wait` gives this reuslt:

```
Worker: 0; Base: 0; Target: 0 (diff: 0)
State: <unknown: 0>; Current Epoch: 0; Todo: 0
Validated 0 messages (0 per second)
```

10.  Hangs forever

**Expected behavior**
Expected `lotus sync wait` to work normally.

**Version (run `lotus version`):**
Build:
```
$ git branch -v
* feat/proofs-5.2.3 8082d3767 Update FFI
  master            7d02cd667 [behind 45] Merge pull request #4649 from filecoin-project/feat/sectorslist-utils
  v1.1.2            7d02cd667 Merge pull request #4649 from filecoin-project/feat/sectorslist-utils
```

Running:
```
$ lotus version
Daemon:  0.10.2+git.8082d3767.dirty+api0.17.0
Local: lotus version 0.10.2+git.8082d3767.dirty
```

**Additional context**

Env vars used in build:

```
export FIL_PROOFS_USE_GPU_COLUMN_BUILDER=1
export FIL_PROOFS_USE_GPU_TREE_BUILDER=1
export FIL_PROOFS_MAXIMIZE_CACHING=1
export RUSTFLAGS="-C target-cpu=native -g"
export FFI_BUILD_FROM_SOURCE=1
```

**Comment by arajasek** [comment]
Can't open the Slack thread in question, but version 0.10.2 probably isn't gonna be able to sync anything. I'm not sure what the SDR branch in question is.

**Comment by mgoelzer** [comment]
Hi @arajasek, think I've gotten the slack links fixed now.  I included a screenshot below to eliminate ambiguity about which thread.

I think I figured it out and now I am retesting.  I'd bee assuming the branch was rebased on the current master, but I think it's actually a branch off of an earlier version of lotus.  As you point out, a 0.10.2 version of the code is probably not going to be compatible with the current stateroots that I was downloading from AWS.

Trying again now to combine `feat/proofs-5.2.3` lotus-miner and lotus-worker binaries with a v1.1.2 daemon.

<img width="901" alt="Screen Shot 2020-11-02 at 6 52 01 PM" src="https://user-images.githubusercontent.com/225111/97937653-80bc4f00-1d3c-11eb-8b9c-e21b02dc79e6.png">
